shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -450,6 +451,11 @@ - (void)viewWillAppear:(BOOL)animated {
     _engineNeedsLaunch = NO;
   }
 
+  if (!_isAttchedView) {
+    [_engine.get() attachView];
+    _isAttchedView = YES;
+  }
```

Comment:
Do we really need this guard? It seems like if we actually loaded a new instance of the view, we'd want to let the engine know about it to update the surface/a11y bridge.

But we would not want to let it know if the view was actually the same view.

Reply:
I think that we this guard.
Make sure that we don't create another instance of `ios_surface_` in platform_view_ios.mm
I think that this is the same with old logic.

Reply:
That's fine.  If this is a bug it's existing and we can address it separately

Reply:
We should probably FML_DCHECK that owner controller is not nil here though

Reply:
I added FML_DCHECK.
Thanks